### PR TITLE
feat: emoteCount

### LINKED ISSUE #1175: Total emotes per collection
Show in series column total emotes per series

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #1175
- [x] Added column Emotes to reflect total emote count on collection

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

File: components/series/SeriesTable.vue [hunk 1]
```diff
@@ -422,6 +434,7 @@ export default class SeriesTable extends mixins(PrefixMixin) {
         buyHistory: axisLize(
           Object.assign({}, defaultBuyEvents, buyEvents[e.id] || {})
         ),
+        emoteCount: e.emoteCount,
```
Comment (kkukelka) [inline components/series/SeriesTable.vue:437]: do you by any chance now it this will always be set? meaning that if no emotes have been applied, is 0 from subqsuid? otherwise we should do something like this 
```suggestion
        emoteCount: e.emoteCount || 0,
```
otherwise `<template>`will crash

**Review by kkukelka — APPROVED** [review]
lgtm but 1 question

File: components/series/SeriesTable.vue (hunk 1 above)
Comment (petersopko) [reply components/series/SeriesTable.vue:437]: yeah you're right, no emotes returns an empty array

Comment (petersopko) [reply components/series/SeriesTable.vue]: thank you !

**Comment by kkukelka** [comment]
pay 100

**Comment by kkukelka** [comment]
pay 100

**Comment by kkukelka** [comment]
> pay 100

oh no same bug as last time?

**Comment by vikiival** [comment]
pay 100 usd
